packages/typo3-docs-theme/resources/template/inline/textroles/unkown.html.twig

```diff
@@ -0,0 +1,3 @@
+{% apply spaceless %}
```

Comment:
Please make a composer update then rename this template to unknown.html.twig. See https://github.com/phpDocumentor/guides/pull/744

Reply:
Done